### PR TITLE
Populate

### PR DESCRIPTION
**Pull request by LizzAlice**
solves #251

### CODE REVIEW

File: main/management/commands/populate.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        room=Room.objects.order_by('?').first(),
+        group=Devicegroup.objects.order_by('?').first(),
+        department=Department.objects.order_by('?').first()
+    )
+
+
+def generate_buildings(number):
+    print("Generating buildings")
+    word_list = fake.words(number, unique=True)
+    Building.objects.bulk_create(fake_building(word_list[i]) for i in range(number))
```
Comment (xi) [inline main/management/commands/populate.py]: could be simply written as

```
Building.objects.bulk_create(fake_building(word) for word in word_list)
```

File: main/management/commands/populate.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
+
+from django.core.management.base import BaseCommand
+
+from faker import Faker
```
Comment (octomike) [inline main/management/commands/populate.py]: missing from dependencies.txt

File: main/management/commands/populate.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+def generate_department_users():
+    print("Generating departmentusers")
+    users = Lageruser.objects.all()
+    for user in users:
+        DepartmentUser.objects.create(user=user, department=user.main_department)
+
+
+def generate_devicetypes(number):
+    print("generating devicetypes")
```
Comment (octomike) [inline main/management/commands/populate.py]: typo

File: main/management/commands/populate.py [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    Room.objects.bulk_create(fake_room(word_list[i]) for i in range(number))
+
+
+def generate_manufacturers(number):
+    print("Generating manufacturers")
+    Manufacturer.objects.bulk_create(fake_manufacturer() for i in range(number))
+
+
+def generate_departments(number):
+    print("Generate departments")
```
Comment (octomike) [inline main/management/commands/populate.py]: typo

**Comment by octomike** [comment]
Cool stuff.

Can you please merge this with and then deprecate `devices/management/commands/createexampledata.py`?

**Comment by octomike** [comment]
also, some lendings, belongings, templates and maybe also images would be nice

**Pull request merged**